samples/gen-js/index.js

```diff
@@ -88,15 +113,25 @@ module.exports = class SwaggerTest {
         }
       }
 
-      request(requestOptions, (err, response, body) => {
-        if (err) {
-          return rejecter(err);
-        }
-        if (response.statusCode >= 400) {
-          return rejecter(new Error(body));
-        }
-        resolver(body);
-      });
+      const retryPolicy = this.retryPolicy || options.retryPolicy || defaultRetryPolicy;
```

Comment:
I'm assuming by `null` you're trying to communicate "no retries"? If so, I think I'd rather export a `NoRetryPolicy` that you can pass (this is what I do with the Go client)